src/app/globals.css

```
@@ -1,11 +1,18 @@
 @import 'tailwindcss';
 
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)